test/addons-napi/test_array/test.js

```diff
@@ -19,19 +19,30 @@ const array = [
   ]
 ];
 
-assert.strictEqual(test_array.Test(array, array.length + 1),
-                   'Index out of bound!');
+assert.throws(
+  () => {
```

Comment:
I was keeping it consistent with what I saw most often in tests.  For example: test/parallel/test-punycode.js.